### PR TITLE
Like functionality

### PR DESCRIPTION
**Pull request by GreyMatteOr**
### Description

Hook-up GraphQL requests in the webapge. Now points at `heroku`!
### Related Issue
close #17 
close #21 

### How Has This Been Tested?

check console logs in browser when clicking like buttons or signup of a user

run `npm test` in the terminal

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/Scss/Make_Post.scss [hunk 1]
```diff
@@ -82,8 +82,8 @@
 }
 
 .tab-links-selected {
-  background-color: $application-bue;
```
Comment (MichaelEWalker87) [inline src/Scss/Make_Post.scss:85]: Should we ignore the CSS files?

**Review by MichaelEWalker87 — COMMENTED** [review]
We are in discussion about what to do on the SCSS file changes. It is possible with the various changes that we will run into a ton of conflicts

**Review by MichaelEWalker87 — APPROVED** [review]
Removed the SCSS Files due to changes coming in the next PR

File: src/Scss/Make_Post.scss (hunk 1 above)
Comment (GreyMatteOr) [reply src/Scss/Make_Post.scss:85]: Yes

**Pull request merged**